### PR TITLE
New recipe for pycapnp

### PR DESCRIPTION
**Pull request by frol**

### CODE REVIEW

**Comment by frol** [comment]
@jakirkham @ocefpaf Do you happen to know what is going on with Linux builds? I'm getting "no source" message on the CI while it builds just fine on my local machine using the same recently released conda-build 1.20.2 and also 1.20.0.

**Comment by ocefpaf** [comment]
No idea @frol. I re-started `CircleCI`, let's hope the failure goes away :wink:

**Comment by frol** [comment]
@ocefpaf I have already retried myself by rebasing and pushing with force and your retry haven't helped either :(

BTW, Linter went crazy also...

File: recipes/pycapnp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - python
+
+test:
+  imports:
+    - capnp
+
+about:
+  home: http://jparyani.github.io/pycapnp/
+  license: BSD 2-clause
+  summary: A python wrapping of the C++ implementation of the Cap’n Proto library
```
Comment (ocefpaf) [inline recipes/pycapnp/meta.yaml]: To "fix" the linter you should change `’` to `'` and add wrap that in `""`. However, to truly fix the linter, we should change it to parse `unicode` :smile:

**Comment by frol** [comment]
I don't know why, but Travis builds Lua before my recipe O_o

```
Resolved dependencies, will be built in the following order: 
	lua-5.1.5-0 (will be built: True)
	pycapnp-0.5.7-py27_0 (will be built: True)
	pycapnp-0.5.7-py34_0 (will be built: True)
	pycapnp-0.5.7-py35_0 (will be built: True)
```

**Comment by ocefpaf** [comment]
> I don't know why, but Travis builds Lua before my recipe O_o

That is [this](https://github.com/conda-forge/conda-forge.github.io/issues/135) problem. Working on it.

**Comment by frol** [comment]
Cool! Linter is happy! :)

**Comment by frol** [comment]
All green! Please, review!

**Comment by pelson** [comment]
> BTW, Linter went crazy also...

What did you do!? 😄 
Seriously though, if you think you have a recipe which caused this, try running ```conda smithy recipe-lint /path/to/recipe``` to confirm, then raise at https://github.com/conda-forge/conda-smithy.

Thanks @frol. (All the other issues should also have now been fixed)

**Comment by frol** [comment]
Why the repo is empty: https://github.com/conda-forge/pycapnp-feedstock?

**Comment by jakirkham** [comment]
Feedstocking is failing. So some things are in an incomplete state. See this issue ( https://github.com/conda-forge/staged-recipes/issues/619 ).

**Comment by jakirkham** [comment]
Should be in working order again. Please let me know if there are other issues.

**Comment by frol** [comment]
Yeap, everything is fine now. Thanks!

**Pull request merged**